cdc/server_test.go

```diff
@@ -15,27 +15,38 @@ package cdc
 
 import (
 	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"net/http"
 	"net/url"
 	"os"
 	"os/user"
 	"path/filepath"
 	"runtime"
+	"sync"
 	"testing"
 	"time"
 
-	"github.com/pingcap/check"
+	"github.com/pingcap/tidb/br/pkg/httputil"
+	"github.com/pingcap/tiflow/cdc/capture"
 	"github.com/pingcap/tiflow/cdc/model"
 	"github.com/pingcap/tiflow/pkg/config"
+	cerrors "github.com/pingcap/tiflow/pkg/errors"
 	"github.com/pingcap/tiflow/pkg/etcd"
+	"github.com/pingcap/tiflow/pkg/retry"
+	security2 "github.com/pingcap/tiflow/pkg/security"
 	"github.com/pingcap/tiflow/pkg/util"
-	"github.com/pingcap/tiflow/pkg/util/testleak"
 	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"github.com/tikv/pd/pkg/tempurl"
 	"go.etcd.io/etcd/clientv3"
 	"go.etcd.io/etcd/embed"
 	"golang.org/x/sync/errgroup"
 )
 
 type serverSuite struct {
+	suite.Suite
```

Comment:
`testify/suite` is not recommended to use. Ref: https://github.com/pingcap/tiflow/issues/2164

Reply:
done